client/package.json

```diff
@@ -17,6 +20,7 @@
     "react-redux": "^7.1.0",
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.0.1",
+    "react-textarea-autosize": "^7.1.0",
```

Comment:
What does it give u?

Reply:
It is a very good lib, it improves optimization for textarea, really useful. Autosize textarea when i need this, it is visible in ui.